tests/x11regressions/firefox/sle12/firefox_flashplayer.pm

```diff
@@ -28,12 +28,17 @@ sub run() {
     assert_screen('firefox-flashplayer-verify_loaded', 45);
 
     send_key "pgdn";
-    assert_screen('firefox-flashplayer-verify', 25);
-
-    send_key "esc";
-    send_key "alt-d";
-    type_string "https://www.youtube.com/watch?v=Z4j5rJQMdOU\n";
-    assert_screen('firefox-flashplayer-video_loaded', 45);
+    # flashplayer dropped since sled12 sp2
+    while (check_screen([qw/firefox-flashplayer-dropped firefox-flashplayer-verify/])) {
```

Comment:
I think I don't get what you exactly mean :(

on sled12 sp2, flashplayer dropped and page shows like this:
https://openqa.suse.de/tests/279168/modules/firefox_flashplayer/steps/9

and on pre-sp2, flashplayer exist and page shows its version info like this:
http://147.2.207.208/tests/797/modules/firefox_flashplayer/steps/7